cpp/CMakeLists.txt

```diff
@@ -530,6 +532,7 @@ target_include_directories(
   cudf
   PUBLIC "$<BUILD_INTERFACE:${DLPACK_INCLUDE_DIR}>"
          "$<BUILD_INTERFACE:${JITIFY_INCLUDE_DIR}>"
+         "$<BUILD_INTERFACE:${KvikIO_INCLUDE_DIR}>"
```

Comment:
Instead of this we should link to the `kvikio::kvikio` target, and remove `KvikIO_INCLUDE_DIR`

Reply:
Done in https://github.com/rapidsai/cudf/pull/10593/commits/bcb0a8ac0a53d551af25a002313b09e91edff887, is this sufficient? 
Can I ask you to take a look at KvikIO's [CMakefile](https://github.com/rapidsai/kvikio/blob/branch-22.06/cpp/CMakeLists.txt) also and see if I am doing the right thing? I am still a novice when it comes to cmake :)